internal/proxy/proxy.go

```diff
@@ -510,20 +510,20 @@ func NewClient(ctx context.Context, d cloudsql.Dialer, l cloudsql.Logger, conf *
 	for _, inst := range conf.Instances {
 		m, err := c.newSocketMount(ctx, conf, pc, inst)
 		if err != nil {
-			for _, m := range mnts {
-				mErr := m.Close()
-				if mErr != nil {
-					l.Errorf("failed to close mount: %v", mErr)
-				}
+			c.logger.Errorf("[%v] Unable to mount socket: %v", inst.Name, err)
+			if getNetworkType(conf, inst) == "unix" {
+				continue
 			}
-			return nil, fmt.Errorf("[%v] Unable to mount socket: %v", inst.Name, err)
 		}
 
 		l.Infof("[%s] Listening on %s", inst.Name, m.Addr())
 		mnts = append(mnts, m)
 	}
 	c.mnts = mnts
-	return c, nil
+	if len(mnts) > 0 {
```

Comment:
Let's swap this with the error below, so the error path is indented.

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Fixed. Thanks!